tests/management/workspace/test_view.py

```diff
@@ -486,7 +487,7 @@ def test_update_workspace_unauthorized(self):
         self.assertEqual(status_code, 403)
         self.assertEqual(response.get("content-type"), "application/problem+json")
 
-    def test_edit_workspace_not_allowed_type(self):
+    def test_edit_workspace_disregard_type(self):
```

Comment:
Proposing we update the intent of this test. As long as we don't publish `type` as a supported field in the spec, we'll just let the serializer (with `read_only=True`) handle this. It won't raise an exception if a value is supplied on create or update, it disregards it, like any other field that's not supported.

Reply:
@astrozzc

Reply:
Good idea